src/components/MenuPanel/OrganizationSwitcher/OrganizationSwitcher.js

```diff
@@ -65,8 +66,7 @@ const OpenButton = styled.button.attrs({ type: 'button' })`
   background: none;
   cursor: pointer;
   padding: 0;
-  margin-left: -20px;
-  width: 180px;
+  width: 100%;
```

Comment:
The idea was to leave the remaining space for the active area of the notifications panel… but we can change it again later. 👍

Reply:
Ahh oops 🙈. Will revert this!